### ISSUE TITLE
FR: Add easy multi-category user filter to list views

### ISSUE THREAD

**Issue by Hoffi1** [jemproject/JEM-Project#1067]
On calendar views are buttons to switch on/off events relative to categories.
Something like this would be useful on list views too.

The mechanism can't be copied because we can't cascade divs. (independent from the fact that cascading is wrong, it should be OR, not AND)

Also the number of events shown on the page should be constant (as selected in dropdown box) so it seems we must send a request to get the right events each time multi-category filter is changed (or a "Go" has clicked).

-- I hope it can be added to next version...

**Comment by mckillo** [comment]
Yes. From JEM 4.4.2 already has multi-category filtering for list views, but it depends on the view.

From the administrator side, in the menu item settings for the Events List view, you can configure a category filter. You can enter several category IDs separated by commas, choose whether those categories should be included or excluded, and optionally include their subcategories too.

Example:

Categories: 3,8,12
Mode: Include
Include subcategories: Yes
This means the frontend list will show events from categories 3, 8, and 12, plus their child categories.

From the frontend user side, this is not a dynamic multi-select filter that visitors can change on the page. The filter is configured by the site administrator in the backend menu item settings. Visitors simply see the resulting filtered list.

For the Category view, it works differently: the menu item points to one category, but there is an option to include events from its subcategories.
